readme.md

```diff
@@ -18,6 +18,25 @@ var ghUser = require('gh-user');
 ghUser('sindresorhus').then(function (user) {
 	console.log(user);
 	//=> { login: 'sindresorhus', id: 170270, ... }
+
+	// The rateLimit property is neither configurable nor enumerable
+	// and doesn't clobber the user object.
+  // You can still access its properties as following:
+	console.log(user.rateLimit['github-time']);
```

Comment:
Instead of logging each property, just do one `console.log(user.rateLimit);` and show the output. We can just link to https://developer.github.com/v3/rate_limit/ for more info.